Example/Backpack/View Controllers/SettingsViewController.swift

```diff
@@ -21,17 +21,95 @@ import Backpack
 
 class SettingsViewController: UITableViewController {
     @IBOutlet weak var closeButton: UIBarButtonItem!
+    @IBOutlet weak var enableThemeSwitch: Switch!
+    var showThemeList: Bool = false
+    @IBOutlet var selectableCells: [UITableViewSelectableCell]!
+
+    enum Theme: Int {
```

Comment:
If this was not scoped here we could have our wrapper around `NSUserDefaults` return an a variant